### ISSUE TITLE
remote desktop port

### ISSUE THREAD

**Issue by lucserre** [OpenMediaVault-Plugin-Developers/openmediavault-virtualbox#21]
When starting a second virtual machine, it assigns remote desktop (vnc) port that is already taken by other running machine.  Both are set to use port range 9000-9100.  First machine starts and gets port 9000 and it works, but then second machine also says 9000 when you start it, but it obviously doesn't connect.  9001 doesn't work either in this situation.

**Comment by ryecoaaron** [comment]
I just tried this on my OMV 4.x test machine and it is working just fine.  What version of OMV are you using?

**Comment by lucserre** [comment]
I use version: 4.1.13-1
Here are screenshots of it happening.
![image](https://user-images.githubusercontent.com/10817213/48383320-4078e780-e6b3-11e8-9d69-4fc1af39da37.png)
![image](https://user-images.githubusercontent.com/10817213/48383344-55557b00-e6b3-11e8-901b-c6049bf8ec4a.png)

**Comment by ryecoaaron** [comment]
Not sure why that is happening on your system but not mine.  I just updated the phpvirtualbox package to 5.2-1 (from 5.2-0).  Making that will fix it?  Not much we can do with the plugin to fix this since this is a problem with phpvirtualbox and/or virtualbox itself.  As a workaround, did you try using a static port instead of the port range?

**Comment by ryecoaaron** [comment]
Actually, there are four packages involved with OMV - virtualbox, phpvirtualbox, virtualbox-ext-pack-installer, and openmediavault-virtualbox

I create the packages for OMV for phpvirtualbox (I don't maintain phpvirtualbox code though).  I noticed that 5.2-1 was out on their github repo.  So, I created a new package for 5.2-1 and put it in the OMV repo.  So, if you go to the Updates tab and click Check, it should show up for you to update your system.
